docs/_docs/tools/control-script.adoc

```diff
@@ -913,3 +915,35 @@ TRANSACTIONS                   SYS       Running transactions
 CACHES                         SYS       Caches
 TASKS                          SYS       Running compute tasks
 Command [SYSTEM-VIEW] finished with code: 0
+
+
+== Performance Statistics
+
+Ignite provides a built-in tool for cluster profiling.
```

Comment:
Lets add link to page: `Read link:monitoring-metrics/performance-statistics[Performance Statistics] for more information.`

Reply:
done